forge/ee/lib/billing/trialTask.js

```diff
@@ -59,12 +55,7 @@ module.exports.init = function (app) {
         for (const subscription of pendingEmailSubscriptions) {
             const endingInDurationDays = Math.ceil((subscription.trialEndsAt - Date.now()) / ONE_DAY)
             const endingInDuration = endingInDurationDays + ' day' + ((endingInDurationDays !== 1) ? 's' : '')
-            const billingUrl = `${app.config.base_url}/team/${subscription.Team.slug}/billing`
-            await sendTrialEmail(subscription.Team, 'TrialTeamReminder', {
-                endingInDuration,
-                billingSetup: subscription.isActive(),
-                billingUrl
-            })
+            const billingUrl = `${app.config.base_url}/team/${subscription.Team.slug}/billing` 
```

Comment:
```suggestion
```

This line is now redundant

Reply:
As above, since the code removed leaves an unused variable, the variable should be deleted too.